google-services.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
I believe this is for non-managed mobile applications. Check [this](https://medium.com/@denisceric94/expo-and-firebase-authentication-email-password-facebook-and-google-681a511c61b6) out